### ISSUE TITLE
AppHeader Styling Updates

### ISSUE THREAD

**Issue by andycwilliams** [codeforpdx/PASS#110]
Styling AppHeader with MUI

**Comment by leekahung** [comment]
Ideally we'll replace AppHeader with the Navbar that's was being made in MUI after review and merge it into Development.

**Comment by xscottxbrownx** [comment]
Agree with @leekahung.

It should be same `<Navbar>` sitewide.  Of course the Navbar will need conditional rendering of what it contains.  Need an `isLoggedIn` state (which I think we already have somewhere - may need to lift the state up.)

and just have something in `<Navbar>` like:
```js

{ isLoggedIn 
? (
   <Typography>PASS</Typography>
   <Link1 />
   <Link2 />
   <IconButton1 />
   <IconButton2 />
   <IconButton3 />
) : (
   <Typography>PASS</Typography>
   <LoginButton />
)
}
```

**Comment by xscottxbrownx** [comment]
Which opens up the `Login.jsx` to being more of an actual landing page for the site. Needing a hero image and at least a description of what PASS is and how it benefits users.    See my previous comments on #109 for same concept.

**Comment by xscottxbrownx** [comment]
Closed with merging of PR #116 

can open new issues for navbar if needed.
